go-controller/pkg/metrics/ovs.go

```diff
@@ -1231,6 +1231,15 @@ func RegisterOvsMetrics() {
 		componentCoverageShowMetricsMap[ovsVswitchd] = ovsVswitchdCoverageShowMetricsMap
 		registerCoverageShowMetrics(ovsVswitchd, MetricOvsNamespace, MetricOvsSubsystemVswitchd)
 
+		prometheus.MustRegister(prometheus.NewProcessCollector(prometheus.ProcessCollectorOpts{
+			PidFn:     prometheus.NewPidFileFn("/var/run/openvswitch/ovs-vswitchd.pid"),
```

Comment:
@girishmg are these pidfile locations the same for you? I think unless somebody builds a custom OVS they should be the same everywhere, but not sure we need/want to support custom OVS paths. We could do config options for that if needed though.

Reply:
@dcbw apologies for the delayed reply.. yes, our OVS uses `/var/run/openvswitch` for pidfiles. 

yes, if needed we can do config options.

we already do this in `ovnkube.sh`. So, in future we can expose the RUN directories for OVS and OVN.
```
# Determine the ovn rundir.
if [[ -f /usr/bin/ovn-appctl ]]; then
  # ovn-appctl is present. Use new ovn run dir path.
  OVN_RUNDIR=/var/run/ovn
  OVNCTL_PATH=/usr/share/ovn/scripts/ovn-ctl
  OVN_LOGDIR=/var/log/ovn
  OVN_ETCDIR=/etc/ovn
else
  # ovn-appctl is not present. Use openvswitch run dir path.
  OVN_RUNDIR=/var/run/openvswitch
  OVNCTL_PATH=/usr/share/openvswitch/scripts/ovn-ctl
  OVN_LOGDIR=/var/log/openvswitch
  OVN_ETCDIR=/etc/openvswitch
fi
```

Reply:
@girishmg OK to merge for now and add config options later if we find somebody using non-standard paths?

Reply:
ACK